R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -67,6 +67,8 @@ sparkSession <- if (windows_with_hadoop()) {
     sparkR.session(master = sparkRTestMaster, enableHiveSupport = FALSE)
   }
 sc <- callJStatic("org.apache.spark.sql.api.r.SQLUtils", "getJavaSparkContext", sparkSession)
+# materialize the catalog implementation
+listTables()
```

Comment:
we are calling `sparkR.session(master = sparkRTestMaster, enableHiveSupport = FALSE)` is almost every other test files - does the same apply in those other places?

Reply:
`test_sparkSQL.R` is the only one uses `newJObject("org.apache.spark.sql.hive.test.TestHiveContext", ssc, FALSE)` on the `ssc`, so the catalog impl spark conf is changed. So ``test_sparkSQL.R` is the only one broken.

Reply:
ok